wallet/subnet/primary/examples/add-permissioned-subnet-validator/main.go

```diff
@@ -60,9 +60,13 @@ func main() {
 	pWallet := wallet.P()
 
 	addValidatorStartTime := time.Now()
+	shortNodeID, err := ids.ShortNodeIDFromNodeID(nodeID)
+	if err != nil {
+		panic(err)
```

Comment:
Can we use `log.Fatal` here like the other errors

Reply:
Done